gatsby-config.js

```diff
@@ -31,6 +31,14 @@ module.exports = {
       resolve: 'gatsby-source-strapi',
       options: strapiConfig,
     },
+    {
+      resolve: 'gatsby-plugin-robots-txt',
+      options: {
+        host: 'https://jalantechnologies.com',
+        sitemap: 'https://www.jalantechnologies.com/sitemap.xml',
```

Comment:
why does sitemap has www. but host does not?

Reply:
"host" refers to domain name of the website, which is main url used to access our site.It's possible to configure a domain name to work with or without the "www." prefix.

Reply:
Noted.

Why is it different here - is there a reason?